ProjectTemplates/templates/ComponentsWebAssembly-CSharp/.template.config/template.json

```diff
@@ -205,11 +205,56 @@
           ]
         },
         {
-          "condition": "(!IndividualLocalAuth || !Hosted)",
+            "condition": "(!IndividualLocalAuth || !Hosted)",
+            "exclude": [
+                "Server/Areas/Identity/Pages/Shared/_LoginPartial.cshtml",
+                "Server/Controllers/OidcConfigurationController.cs",
+                "Server/Models/ApplicationUser.cs"
+            ]
+        },
+         {
+          "condition": "(!GenerateApi)",
+            "exclude": [
+              "Client/Pages/CallWebApi.razor",
+              "Server/Controllers/CallWebApiController.cs"
+            ]
+        },
+        {
+          "condition": "(!GenerateGraph)",
+          "exclude": [
+            "Client/Shared/NavMenu.CallsMicrosoftGraph.razor",
+            "Client/Pages/ShowProfile.razor",
+            "Server/Controllers/ShowProfileController.cs"
+            ]
+        },
+        {
+          "condition": "(!GenerateApiOrGraph)",
+          "rename": {
+            "Client/Shared/NavMenu.NoGraphOrApi.razor": "Client/Shared/NavMenu.razor"
```

Comment:
In **blazorwasm2-singleorg-hosted.Client**, under Client\Shared we don't have the **NavMenu.razor** but the **NavMenu.NoGraphorAPI.razor.razor**

My hypothesis is that the template engine did not do the work correctly because the file is named NavMenu.NoGraphorAPI.razor :( on disk [note the or instead of Or)

See below **ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Client/Shared/NavMenu.NoGraphorApi.razor**

Because on Windows the file name are case insensitive, to fix this, you would need to send 2 commits:
- one commits that deletes the NavMenu.NoGraphorApi.razor file
- another commit to re-add it.
The best would be to move it to another folder do the first commit, rename it in the other folder, and move it back where it should be.

Reply:
code flow is not opening for me atm. but i addressed this.